src/drivers/ports/esp/exti_port.i

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
A general comment: Always add the suffix ``_p`` to pointers to make the code easier to read.